supertokens_python/supertokens.py

```diff
@@ -292,8 +292,8 @@ def get_all_cors_headers(self) -> List[str]:
     async def get_user_count(  # pylint: disable=no-self-use
         self,
         include_recipe_ids: Union[None, List[str]],
+        user_context: Optional[Dict[str, Any]],
```

Comment:
user_context should be last arg. This is an internal function so you can remove the `= None` from the tenant_id, and wherever you are calling this function, you can pass in None by default.

Reply:
This applies to all the changes in this file.